workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -2,35 +2,31 @@
 
 describe Lita::Handlers::Teamcity, lita_handler: true do
   before do
-    registry.config.handlers.teamcity.site             = 'https://build.ankicore.com'
+    registry.config.handlers.teamcity.site             = 'https://site.teamcity.com'
     registry.config.handlers.teamcity.username         = 'username'
     registry.config.handlers.teamcity.password         = 'password'
-    registry.config.handlers.teamcity.git_uri          = 'git@github.com:anki'
+    registry.config.handlers.teamcity.git_uri          = 'git@github.com:git_url'
```

Comment:
Should be : 'git@github.com:git_uri' to match with registry.config.handlers.teamcity.git_uri